src/Relay/Queries/ArtistSearch.ts

```diff
@@ -0,0 +1,18 @@
+import * as Relay from "react-relay"
+
+export class ArtistSearchQueryConfig extends Relay.Route {
+  public static queries = {
+    viewer: (component, params) => Relay.QL`
+      query {
+        viewer {
```

Comment:
> As a convention, Facebook’s model code always accepts an object called the viewer, which represents the entity currently asking for the data.

Using the `viewer` as the root field is what enabled us to get around relay not allowing us to return an array/list of artists. Here is more [context around this issue](https://github.com/facebook/relay/issues/112#issue-101726565).